CodeStarterCamp_Week1/main.swift

```diff
@@ -1,27 +1,41 @@
 //
-//  main.swift
+//  STEP3.swift
 //  CodeStarterCamp_Week1
 //
-//  Created by yagom.
-//  Copyright © yagom academy. All rights reserved.
+//  Created by Jay Shin on 2/12/24.
 //
 
 import Foundation
 
-func drawIceCreamBody(bodyWidth: Int, bodyHeight: Int) {
-    for _ in 1...bodyHeight {
-        for _ in 1...bodyWidth {
-            print("*", terminator: "")
-        }
-        print("")
+drawCookie(cookieLength: 10, cookieBase: "***", cookieTopping: " ", stickLength: 4)
+drawCookie(cookieLength: 12, cookieBase: "***", cookieTopping: "&", stickLength: 4)
+drawCookie(cookieLength: 12, cookieBase: "***", cookieTopping: "#", stickLength: 6)
+drawCookie(cookieLength: 6, cookieBase: "|0|", cookieTopping: " ", stickLength: 4)
+
+func drawCookie(cookieLength: Int, cookieBase: String, cookieTopping: String, stickLength: Int) {
+    print("""
+<정보>
+길이: \(cookieLength)
+몸통: \(cookieBase)
+토핑: \(cookieTopping)
+막대길이: \(stickLength)
+""")
```

Comment:
```suggestion
    print("""
          <정보>
          길이: \(cookieLength)
          몸통: \(cookieBase)
          토핑: \(cookieTopping)
          막대길이: \(stickLength)
          """)
```

여러줄 문자열은 닫는 큰 따옴표의 위치를 기반으로 책정되기 때문에 현재의 방식도 상관은 없습니다.

그런데 저는 여는 큰 따옴표와 위치를 맞추는 것이 읽기 쉬운 것 같습니다. 다만 꼭 그래야하는 것은 아닙니다. 😃

Reply:
본문 내용을 들여쓰기하여 정렬하는게 가독성에 좋은 것 같습니다.
의견주신 내용처럼 수정했습니다.